src/components/structures/TabbedView.tsx

```diff
@@ -118,13 +119,23 @@ export default class TabbedView extends React.Component<IProps, IState> {
     public render(): React.ReactNode {
         const labels = this.props.tabs.map(tab => this._renderTabLabel(tab));
         const panel = this._renderTabPanel(this.props.tabs[this._getActiveTabIndex()]);
-
+        const logoUrlSecondary = customConfig.logoUrlTabbedViewFooter;
+        const logoUrlAltText =  customConfig.logoDescTabbedViewFooter;
         return (
             <div className="mx_TabbedView">
                 <div className="mx_TabbedView_tabLabels">
                     {labels}
                 </div>
                 {panel}
+                {customConfig.tabbedViewdisplaySecondaryLogo === true &&
+                    <div className="mx_AuthFooter_TabbedView">
+                        <a href="" target="_blank" rel="noreferrer noopener">                        
```

Comment:
What is this empty hyperlink for?  If it is no longer required pls remove it

Reply:
Sure

Reply:
done